frontend/src/modules/app/ToolBar.tsx

```diff
@@ -132,8 +155,13 @@ const alarmConfiguration = (ventilationMode: VentilationMode | null): Array<Alar
 /**
```

Comment:
(line 138) @rohanpurohit Did you copy-paste this code from `modules/alarms/AlarmsPage.tsx`? If so, it'd be better just import the interface from there instead, for maintainability & commonality.

Reply:
also fixed in 362